getting_started/build-guide/build-from-scratch.rst

```diff
@@ -12,40 +12,41 @@ Intel platforms include: |BYT|, |CHT|, |HSW|, |BDW|, |APL|, |CNL|, |ICL|, |JSL|,
 
 Support also exists for NXP i.MX8/i.MX8X/i.MX8M platforms.
 
-Build SOF
-*********
-
 The following steps describe how to install the SOF development
 environment on Ubuntu 16.04, 18.04, 18.10, and 20.04. They should work on
 19.04, 19.10 and other Linux distributions with minor or no
 modifications.
 
 .. note::
 
-   Point the ``$SOF_WORKSPACE`` environment variable to the directory in
-   which you store all sof work.
+   Building |SOF| from scratch might take several hours. We recommend
+   that you use Docker to build SOF. For more information, see
+   :ref:`build-with-docker`.
```

Comment:
That's oversimplified and too negative, gives a bad first impression. It sounds like a one-line change in a some `sof/common.h` file might take several hours where in reality that costs less than a minute to rebuild all platforms. There is only a single step that takes hours and that's building the toolchains and it's required only once and then never again.

The problem is the name of this section. It's technically not "Build SOF from scratch" but "Build everything from scratch". Can you change that section name?

Then the start of this paragraph can be changed to "Building the toolchains from scratch might take..."

Reply:
Or a better section name would be to 'build the development environment from sources', ''set up the development environment from sources' or along these lines.